### PR TITLE
@broskoski => Add back doge mode easter egg

### PR DESCRIPTION
**Pull request by mzikherman**
still relevant and funny!

screen shot 2016-08-18 at 12 38 46 pm

### CODE REVIEW

**Comment by broskoski** [comment]
oops, conflicts!

**Comment by mzikherman** [comment]
💚

File: apps/artwork/components/doge/query.coffee [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module.exports = """
```
Comment (broskoski) [inline apps/artwork/components/doge/query.coffee]: 🎨

**Comment by broskoski** [comment]
💯

**Comment by orta** [comment]
10/10

**Comment by zamiang** [comment]
Yayyyy!!! OMG so excited this is back.

**Pull request merged**